diff --git a/client/main.c b/client/main.c
index 24741a3..0744e79 100644
--- a/client/main.c
+++ b/client/main.c
@@ -29,7 +29,7 @@ struct client_stats {
 
 void * client(void * arg);
 const double Z = 1.96; // 95% probability estimated value
-const double E = 5000; // lies within +/- 5us of true value
+const double E = 100000; // lies within +/- 100us of true value
 const int MAX_ITERATIONS_PER_ROUND = 1000000;
 const int MAX_SPIN_SECONDS = 3*60;
 static const int ONE = 1;
